<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8C0LO3K" failures="1" tests="8" name="Test" time="43.603" errors="0" timestamp="2024-03-15T20:39:20 IST">
  <testcase classname="com.obsqura.tests.ExpenseRecordTest" name="verifyUserIsAbleTocreateExpenseRecord" time="5.928"/>
  <testcase classname="com.obsqura.tests.LoginTest" name="verifyUserIsAbleToLogin" time="0.801"/>
  <testcase classname="com.obsqura.tests.CreateDeleteExpenseTest" name="verifyUserIsAbleTocreateDeleteExpense" time="3.309">
    <skipped/>
  </testcase> <!-- verifyUserIsAbleTocreateDeleteExpense -->
  <testcase classname="com.obsqura.tests.CreateDeleteExpenseTest" name="verifyUserIsAbleTocreateDeleteExpense" time="2.556">
    <failure type="java.lang.AssertionError" message="expected [Expense Category Deleted ] but found [Expense Category Deleted Successfully]">
      <![CDATA[java.lang.AssertionError: expected [Expense Category Deleted ] but found [Expense Category Deleted Successfully]
at com.obsqura.tests.CreateDeleteExpenseTest.verifyUserIsAbleTocreateDeleteExpense(CreateDeleteExpenseTest.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserIsAbleTocreateDeleteExpense -->
  <testcase classname="com.obsqura.tests.CreateDeliveryBoyTest" name="createDeliveryBoy" time="3.803"/>
  <testcase classname="com.obsqura.tests.CreateDeliveryBoyTest" name="createDeliveryBoy" time="3.014"/>
  <testcase classname="com.obsqura.tests.CreateNewsTest" name="createNews" time="2.097"/>
  <testcase classname="com.obsqura.tests.AdminTest" name="createAdmin" time="2.751"/>
</testsuite> <!-- Test -->
